datasets/ukb-ppp.yaml

```diff
@@ -0,0 +1,38 @@
+Name: UK Biobank Pharma Proteomics Project (UKB-PPP)
+Description: The UKB-PPP is a collaboration between the UK Biobank (UKB) and thirteen
+  biopharmaceutical companies characterising the plasma proteomic profiles of
+  54,219 UKB participants. As part of a collaborative analysis across the
+  thirteen UKB-PPP partners, we conducted comprehensive protein quantitative
+  trait loci (pQTL) mapping of 2,923 proteins that identifies 14,287 primary
+  genetic associations, of which 85% are newly discovered, in addition to
+  ancestry-specific pQTL mapping in non-Europeans. We identify independent
+  secondary associations in 87% of cis and 30% of trans loci, expanding the
+  catalogue of genetic instruments for downstream analyses.
+Documentation: https://www.synapse.org/#!Synapse:syn51364943/
+Contact: matthias.arnold@helmholtz-munich.de
+ManagedBy: Sage Bionetworks
+UpdateFrequency: Never
+Tags:
+  - genome-wide association study
```

Comment:
the test failure says.. `Invalid tag! genome-wide association study`.  This `genome-wide association study` is NOT in the [list of valid tags](https://github.com/awslabs/open-data-registry/blob/main/tags.yaml).  You can only put valid tags here.